src/platform/linker/Makefile

```diff
@@ -2,7 +2,7 @@ include Makefile.src
 
 CC=gcc
 LD=ld
-CFLAGS=-m32 -D__x86__ -D_GNU_SOURCE -lpthread -Wall -Wextra -Wno-unused-parameter -Wno-unused-function -Wno-pointer-to-int-cast -Wno-int-to-pointer-cast -Wno-format -ggdb3 -I$(SHAREDINC)
+CFLAGS=-D__x86__ -D_GNU_SOURCE -lpthread -Wall -Wextra -Wno-unused-parameter -Wno-unused-function -Wno-pointer-to-int-cast -Wno-int-to-pointer-cast -Wno-format -ggdb3 -I$(SHAREDINC)
```

Comment:
I doubt if this flag is necessary. Since you never changed the linker and loader code, this makefile should remain what it is as before.

Reply:
This is to make it work on x86-64 systems, right? I thought that the mainline already had that?